web/src/scheduled_messages_ui.js

```diff
@@ -62,10 +62,7 @@ export function open_scheduled_message_in_compose(scheduled_msg, should_narrow_t
         narrow_via_edit_scheduled_message(compose_args);
     }
 
-    compose.clear_compose_box();
-    compose_banner.clear_message_sent_banners(false);
```

Comment:
Via `start` we call this function (without the `false` argument), so this call was overridden anyway.